java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S4508_java.html

```diff
@@ -0,0 +1,19 @@
+<p>Object deserialization from an untrusted source can lead to unexpected code execution. ObjectInputStream doesn't provide a way to apply rules on
+its InputStream argument. Knowing that all serializable classes in the scope of the classloader will be deserialized, there is a possibility to
+execute malicious code during the deserialization phase even if, at the end, a ClassCastException will be raised.</p>
+<p>This rule raises an issue when one of the methods "readArray", "readObject", "readUnshared" is called on a "java.io.ObjectInputStream" or any
```

Comment:
please update rule to not contain "readArray"

Reply:
Done. Thanks!